boards/qn9080dk/include/periph_conf.h

```diff
@@ -28,6 +28,29 @@
 extern "C" {
 #endif
 
+/**
+ * @name ADC configuration
+ *
+ * Names "An" are as described in the "Analog In" header on the PCB. All "An"
+ * inputs are configured referenced to 1.8V.
```

Comment:
Are the ADC pins fixed?

Reply:
Yes, there are 8 ADCn external inputs (plus a reference input) and each one of them is available at only one pin.